update.sh

```diff
@@ -44,7 +44,8 @@ function get_url() {
 
 hub --version &> /dev/null || die "hub is not in PATH. Get it from https://github.com/github/hub"
 
-kbt_in_kong_v=$(curl -sL https://raw.githubusercontent.com/Kong/kong/$version/.requirements | grep 'KONG_BUILD_TOOLS_VERSION\=' | awk -F"=" '{print $2}' | tr -d "'[:space:]")
+#kbt_in_kong_v=$(curl -sL https://raw.githubusercontent.com/Kong/kong/$version/.requirements | grep 'KONG_BUILD_TOOLS_VERSION\=' | awk -F"=" '{print $2}' | tr -d "'[:space:]")
+kbt_in_kong_v=4.33.19
```

Comment:
what is the reason for hardcoding this?

Reply:
This chunk fetches the kbt version used in kong so it always matches. However, the git ref hardcoded in `.requirements` is a branch name, which is causing issues elsewhere. We will revert this soon.